src/main.rs

```diff
@@ -44,6 +44,7 @@ See `cargo metadata --help` for a list of flags it supports.";
 struct Args {
     help: bool,
     command: String,
+    diffable: bool,
```

Comment:
In the short term it only seems to complicate things - e.g. argument parsing. It's also not clear if the future additions would be replacing the diffable format (two incompatible flags) or not, simply because we haven't made them yet.

Besides, argument validation needs a refactoring anyway (see #40) and I don't want to hold up this PR on that.